extjsdk/src/test/java/io/vantiq/extjsdk/TestExtensionWebSocketClient.java

```diff
@@ -174,7 +172,7 @@ public void testQueryError() {
         assert socket.compareData("body.messageCode", errorCode);
     }
     
-    
+    int WAIT_PERIOD = 10; // Milliseconds to wait between checks on async actions
```

Comment:
Might we worth creating an ExtensionTestBase.java or something that has this `waitUntilTrue()` method in it rather than copying.  This _technique_ is used in many of our tests to share things as well as to handle common `setup()`/`cleanup()` stuff.

Reply:
This one will probably go in another PR